mac-setup/build-app.sh

```diff
@@ -16,34 +24,33 @@ cd "${0%/*}" || exit
 
 # delete old app, if there
 echo "clean old app"
-
-export PATH="$HOME/.local/bin:$1/inst/bin:$PATH"
-
 rm -rf ./Xournal++.app
-rm ./Xournal++.zip
 
 echo "prepare gtk-mac-bundler"
 if [ ! -d "gtk-mac-bundler" ]; then
   git clone https://gitlab.gnome.org/GNOME/gtk-mac-bundler.git
-  cd gtk-mac-bundler || exit
-else
-  cd gtk-mac-bundler || exit
-  git pull
+  pushd gtk-mac-bundler
+  make install
+  popd
 fi
 
-make install
-cd ..
+# add gtk-mac-bundler to PATH
+export PATH="$PATH":"$HOME"/.local/bin
+
+GTK_MAC_BUNDLER=$(which gtk-mac-bundler)
```

Comment:
Prefer `command -v` (or the usage of any builtin) over `which`

Reply:
This doesn't really matter since this script is assumed to run on macOS, and the `which` executable always exists. `command -v` is not portable across shells.

Reply:
It is `which` that's generally not portable across shells. There are dozens of different implementations, each with different sets of flags and peculiarities; sometimes it's a builtin, other times it's an executable - and it's not in any known standard I know of. `command[ -v]` and its behavior has been clearly specified as a shell builtin in POSIX for decades

I'd say none of my suggestions here "really matter" in the sense that they significantly change the behavior of the script. I suppose I shouldn't have suggested them then, and maybe my bias and personal style bleed through these recommendations. Should probably resign to the fact that these things sneak in

Reply:
I tried to resolve my comments, but it looks like they've already been resolved. Happy to see this improved building experience for Mac